server/chat-plugins/helptickets.js

```diff
@@ -139,6 +171,10 @@ class HelpTicket extends Rooms.RoomGame {
 	 * @param {User} user
 	 */
 	onLogMessage(message, user) {
+		if (!this.ticket.open) return;
+		if (user.isStaff && this.ticket.userid !== user.userid) {
+			if (!this.involvedStaff.has(user.userid)) this.involvedStaff.add(user.userid);
```

Comment:
That's the point of using a Set, so you don't get duplicate values. The only reason you check if the value is already there is if you want to show an error message.
```suggestion
			this.involvedStaff.add(user.userid);
```